playbook-website/app/javascript/components/ReactSidebar/MenuData/GuildesNavItems.ts

```diff
@@ -0,0 +1,18 @@
+export const GuidesNavItems = [
```

Comment:
I've very concerned this is not coming from the back end. Lots of work was put into having this dynamically available. This is available from `DOCS` in the initializer

Reply:
Okay no problem. I'll render it from there.

Reply:
The original reason we were not able to use this was that since this method pulls the info from the url, it was returning null when on on a guides page.....this meant that our sidebar would not populate the nav items under 'getting started' if we were not on a guides/ page.

We did not have this issue when it was built because we had a separate, custom sidebar on the guides pages that only rendered on those pages. 

@jasoncypret 

Would you like me to investigate changing/updating the DOCS area, so that it will work with our pages now in this story? Or save it for a diff story?

Reply:
I copied this over from Nida's sidebar that is on prod. ^^ And that's some of the reasoning.

Reply:
I got it working now with the back end data.